### ISSUE TITLE
2019-11-29 tc010 SD setting mct$TIME to dataset column name and mct$TIMEU no longer works

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#69]
mct$TIME is generally expected to be a value of either "ACTUAL" or "NOMINAL". However, it's permitted to be set directly to a input concentration dataset name such as TIME=="PKATPD". If this is the case another variable, TIMEU, must also be set in the MCT. However, this is no longer working. It does work in the master branch.

So if mct is modified as follows:
```
> mct$TIME <- "PKATPD"
> mct$TIMEU <- "PKATPDU" # need to add TIMEU if TIME points directly to a variable
> summary(d[,mct$TIME])
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   0.00    1.50    8.00   24.93   30.14  120.07 
> table(d[,mct$TIMEU])

 HR 
624 
```

with commit 22d43ea the following error occurs:

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
 Show Traceback
 
 Rerun with Debug
 Error in validate_timeconc_data(map, data, verbose = verbose) : 
  object 'atime' not found 
```

**Comment by rudraya** [comment]
This issue was addressed in 6e99fab, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
This was retested with commit b8deb76 successfully and can be closed.
